spacepy/pybats/ram.py

```diff
@@ -1158,99 +1158,92 @@ def __init__(self, filename, *args, **kwargs):
         self.attrs['file'] = filename
 
         # Read and parse file.
-        f = open(filename, 'r')
-        lines = f.readlines()
-        f.close()
+        with open(filename, 'r') as fh:
+            lines = fh.readlines()
+        head_entries = lines[1].strip().split()
+        variables = head_entries[:-1]
+        p_unit = head_entries[-1].replace('[', '').replace(']', '')
+
+        def name_map(namein):
```

Comment:
Small request: Include, either in comments or docstring, example of an entry to be converted.  Seeing that in-line is super helpful when returning to this code to debug an edge case or new case.